pkg/metrics/upstream.go

```diff
@@ -45,11 +45,13 @@ const (
 	UpstreamRequestPendingOverflow                 = "request_pending_overflow"
 	UpstreamRequestDuration                        = "request_duration_time"
 	UpstreamRequestDurationTotal                   = "request_duration_time_total"
+	UpstreamProxyDuration                          = "proxy_duration_time"
```

Comment:
Could you add some comments to show the difference between request/proxy duration?